templates/includes/cookie-consent-banner.tx

```diff
@@ -37,7 +37,7 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <div class="govuk-cookie-banner__content">
-          <p>You've rejected analytics cookies. You can <a class="govuk-link" href="{{ CHS_URL }}/help/cookies">change your cookie settings</a> at any time.</p>
```

Comment:
The forward slash here has been taken out.

Reply:
@aokunade Thanks Aaishah, just pushed a second commit adding the missing hyphen - thank you